app/views/_mobile-service-instance-row.html

```diff
@@ -0,0 +1,77 @@
+<div class="list-pf-item provisioned-service" ng-class="{ active: row.expanded }">
```

Comment:
The view should be accessible.

Reply:
I'm not sure what I need to change here, could you elaborate?